templates/layouts/master.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
```

Comment:
I forgot to raise this before so sorry for the delay (I thought I did). But there are some missing critical meta tags.

- Never omit the charset as it can break the rendering:

```
<meta charset='utf-8'>
```

Read more about it in [this classic article](https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/).

- The layout is not responsive now as it's missing this meta tag:

```
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Nor the content is just scaled down to the screen size instead of taking the whole page size:

![image](https://user-images.githubusercontent.com/696529/100056777-d88c3880-2e58-11eb-81c6-6a09b14a45b4.png)

The other meta tags can be omitted but these two are definitely compulsory.

Reply:
Thank you so much for the knowledge and resource.
Fixed in 421e7392c42836cc55af919268121a80a6b04ede

Reply:
Let's add some padding on `main` on small screens though now that the screen is resized properly:

![image](https://user-images.githubusercontent.com/696529/100192107-e5299300-2f23-11eb-871d-343d17291e81.png)

Reply:
Added padding in a25b2dc7b29130fe7f4bc9a31a12412f153c480f